.gitlab-ci.yml

```
@@ -1143,11 +1143,15 @@ workflow:
     when: never
   - <<: *if_auto_e2e
 
-.on_omnibus_change:
+.on_packaging_change:
   - !reference [.except_mergequeue] # The prerequisites are not run in the mergequeue pipeline so we need to skip this rule
   - changes:
       paths:
         - omnibus/**/*
+        - .gitlab-ci.yml
+        - .gitlab/package_build.yml
+        - release.json
+        - .gitlab/package_build
```

```suggestion
        - .gitlab/package_build/**/*
```
?

(I'm assuming that is needed, given how the first rule is written, but I haven't checked)

---

You're right, thanks !